routes/data.js

```diff
@@ -248,3 +248,117 @@ exports.organizationsPost = async function (req, res) {
     .update(newOrgData);
   res.redirect('/dashboard');
 };
+
+exports.getFavorites = async function (req, res) {
+  // If user is not authenticated then return no authorized
+  if (!req.user) {
+    res.sendStatus(401);
+    return;
+  }
+
+  const memberQuery = await firestore
+    .collection(resolveCollectionName('Members'))
+    .where('authenticationID', '==', req.user.id)
+    .get();
+  const memberRef = memberQuery.docs[0].ref;
+
+  const favoritesSnapshot = await firestore
+    .collection(resolveCollectionName('Favorites'))
+    .where('member', '==', memberRef)
+    .get();
+  const results = {};
+  favoritesSnapshot.docs.forEach((doc) => {
+    results[doc.id] = doc.data();
+  });
+  res.send(results);
+};
+
+exports.getFavoriteOfMember = async function (req, res) {
```

Comment:
Does this return whether or not a given organization is favorited? If so, it might be clearer to call it `isFavoriteOfMember` to imply a boolean.

Reply:
Also, if the above is true, could we make this a backend function rather than a route response? It could be more efficient to call this method on each organization before sending the organization list to the front-end.